### PR TITLE
Frontend 1657 - Add Synaptic Footer to Dev Center

### PR DESCRIPTION
**Pull request by langlk**
Resolves Dev Center side of [FRONTEND-1657](https://algorithmia.atlassian.net/browse/FRONTEND-1657)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "synapse"]
+	path = synapse
+	url = ../synapse.git
```
Comment (bnlb) [inline .gitmodules]: Thanks for using a relative url here. 💯

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+#
+# Build Synapse dist/ files
+#
+FROM node:10.14 as style-builder
+
+WORKDIR /app
+COPY ./synapse .
+
+# Install dependencies
+RUN npm ci
```
Comment (bnlb) [inline Dockerfile]: I had never heard of this before. Nice find.

File: _layouts/default.html [hunk 3]
```diff
@@ -12,6 +12,8 @@
     {% include open-graph.html %}
     <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/pygments-github.css">
     <link rel="stylesheet" href="{{ site.baseurl }}/css/vendors.css">
+    <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/synapse/dist/scoped-synapse.css">
+    <link rel="stylesheet" href="{{ site.cdnurl }}{{ site.baseurl }}/css/devcenter.css">
```
Comment (bnlb) [inline _layouts/default.html]: We should update these lines to remove `{{ site.cdnurl }}` as they're currently 404ing.

File: _includes/footer.html [hunk 4]
```diff
@@ -1,21 +1,167 @@
...
-    </div>
+<footer role="contentinfo" id="site-footer" class="syn-styles-supported">
+  <div class="syn-container">
+    <div class="syn-row syn-mb-32">
+      <div class="syn-col syn-col-lg-2 syn-col-md-3">
+        <div class="syn-mb-8">
+          <a href="/">
+            {% if site.enterprise %}
+            <img
+              v-else
```
Comment (bnlb) [inline _includes/footer.html]: No need for the `v-else` here since we're outside the context of Vue.

File: _includes/footer.html [hunk 5]
```diff
@@ -1,21 +1,167 @@
...
-        {% endif %}
-        <li><a href="{{ href }}" {% if link.url contains 'http' %}target="_blank"{% endif %}>{{ link.title }}</a></li>
-      {% endfor %}
-      </ul>
-    </nav><!-- /.bottom-menu -->
+        <div class="syn-col syn-col-lg-2 syn-col-md-3">
+          <div class="syn-overline syn-mb-8">Product</div>
+          <div class="syn-mb-8 syn-body-2">
+            <a
+              to="/pricing"
```
Comment (bnlb) [inline _includes/footer.html]: This `to` should be an `href`, and `/pricing` is a web client route, not a dev center route. Once this is a standalone container this relative route will 404. We'll need to ensure all links in the footer that route to the web client are absolute.

File: _includes/footer.html (hunk 4 above)
Comment (langlk) [reply _includes/footer.html]: Removed!

File: _includes/footer.html (hunk 5 above)
Comment (langlk) [reply _includes/footer.html]: Thanks for catching!

File: _layouts/default.html (hunk 3 above)
Comment (langlk) [reply _layouts/default.html]: They are? Are you running the dev config?

Comment (langlk) [reply _layouts/default.html]: Okay, I do see the fonts getting blocked. Updating!

**Comment by langlk** [comment]
Currently deployed to test

**Pull request merged**